cmd/gateway-azure.go

```diff
@@ -457,38 +487,54 @@ func (a *azureObjects) DeleteObject(bucket, object string) error {
 	return nil
 }
 
-// ListMultipartUploads - Incomplete implementation, for now just return the prefix if it is an incomplete upload.
-// FIXME: Full ListMultipartUploads is not supported yet. It is supported just enough to help our client libs to
-// support re-uploads. a.client.ListBlobs() can be made to return entries which include uncommitted blobs using
-// which we need to filter out the committed blobs to get the list of uncommitted blobs.
+// ListMultipartUploads - It's decided not to support List Multipart Uploads, hence returning empty result.
 func (a *azureObjects) ListMultipartUploads(bucket, prefix, keyMarker, uploadIDMarker, delimiter string, maxUploads int) (result ListMultipartsInfo, err error) {
-	result.MaxUploads = maxUploads
-	result.Prefix = prefix
-	result.Delimiter = delimiter
-	meta := a.metaInfo.get(prefix)
-	if meta == nil {
-		// In case minio was restarted after NewMultipartUpload and before CompleteMultipartUpload we expect
-		// the client to do a fresh upload so that any metadata like content-type are sent again in the
-		// NewMultipartUpload.
-		return result, nil
-	}
-	result.Uploads = []uploadMetadata{{prefix, prefix, UTCNow(), "", nil}}
+	// It's decided not to support List Multipart Uploads, hence returning empty result.
 	return result, nil
 }
 
+type multipartMetadata struct {
+	Name     string            `json:"name"`
+	Metadata map[string]string `json:"metadata"`
+}
+
+func getMetadataObjectName(objectName, uploadID string) string {
```

Comment:
rename to getAzureMultipartMetadataPath() so that it is consistent with what it returns (which is azureMultipartMetadata's path)

you can also use getAzureMultipartMetadataObjectName() if you are particular about using `ObjectName` instead of `Path`

Reply:
As this function returns `minio.sys.tmp/multipart/v1/%s.%x/azure.json` which is metadata object name of given object, I think `getAzureMetadataObjectName()` is clear enough